saves/brian.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
you probably don't want to check your savegames in.  Remove this file and add `saves/` to your .gitignore